src/infrastructure/remote/user.ts

```diff
@@ -151,6 +152,70 @@ export function userDataRemote(): UserService {
     return { isSuccess: response.success };
   };
 
+  const getMyAnswerInfo = async (page: number, formID?: number) => {
+    const response = await privateAPI.get({
+      url: formID
+        ? `/form/answer/pick?formId=${formID}&offset=${page}&limit=${PICK_PAGE}`
+        : `/form/answer/pick?offset=${page}&limit=${PICK_PAGE}`,
+    });
+    return {
+      formList: response.data.form
+        ? response.data.form.map((form: any) => ({
+            id: form.formId,
+            profileImage: form.darkIconImage,
+          }))
+        : [],
+      answerList: response.data.answer
+        ? response.data.answer.map((answer: any) => ({
+            id: answer.answerId,
+            formId: answer.formId,
+            icon: answer.darkIconImage,
+            question: answer.title,
+            content: answer.content,
+            isBookmarked: answer.isPinned,
+            keywordList: answer.keywords.map((keyword: any) => ({
+              id: keyword.name,
+              content: keyword.name,
+              color: keyword.colorCode,
+              fontColor: keyword.fontColor,
+            })),
+          }))
+        : [],
+    };
+  };
+
+  const getMyFeedbackInfo = async (page: number, teamID?: number) => {
+    const response = await privateAPI.get({
+      url: teamID
+        ? `/team/feedback/pick?teamId=${teamID}&offset=${page}&limit=${PICK_PAGE}`
+        : `/team/feedback/pick?offset=${page}&limit=${PICK_PAGE}`,
+    });
```

Comment:
필터링 할 팀 아이디 없으면 필터 처리 없이 전부 조회된다고 해서 이렇게 짰는데 맞는 방법인지 궁금합니다..

Reply:
아주 훌륭합니다!!

```suggestion
	const queryParamTeamID = teamID ? `teamId=${teamID}&` : "";
    const response = await privateAPI.get({
      url: `/team/feedback/pick?${queryParamTeamID}offset=${page}&limit=${PICK_PAGE}`
    });
```

정말 줄이고 싶다면 이런 식으로 줄일 수 있을 것 같아요~!

Reply:
이걸로 반영해 보겠습니다 ㅎㅎ